taskcat/project_generator.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+from collections import namedtuple
+from jinja2 import Template
+from os import path
+
+TEMPLATES_ROOT_DIR = 'project_templates'
+TEMPLATE_FILE_EXTENSION = '.jinja'
+
+
+def full_path(root, resource):
+    return root + os.sep + resource
+
+
+def template_paths(template_dir, templates):
+    return [
+        template_dir + os.sep + t
+        for t in templates
+        if t.endswith(TEMPLATE_FILE_EXTENSION)]
+
+
+ProjectConfiguration = namedtuple(
+    'ProjectConfiguration',
+    'owner_email, project_name, project_type, supported_regions'
+)
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
might be better not to hardcode this and let the consumer decide on log level

Reply:
that's fair.